src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends ColorSupplier {
+    private String color;
+    private int number;
+
+    public Ball() {
+        this.number = new Random().nextInt(101);
+        this.color = super.getRandomColor();
+    }
+
+    public String getBall() {
+        StringBuilder builder = new StringBuilder();
+        builder.append(number);
+        String out = color + " " + builder.toString();
+        return out;
```

Comment:
You don't need a string builder here, just return the result of concatenation:

https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method

```java
return "color: " + color + ...;
```